autoscheduler/scraper/models/course.py

```diff
@@ -1,4 +1,18 @@
 from django.db import models
 
-class Course(models.Model):
-    """ Describe model here """
+class Course(models.Model): # Directly from GBS:w

```

Comment:
> # Directly from GBS:w

Did you copy and paste this directly from my model? This comment isn't needed(also includes `:w` from vim lmao)

Reply:
Yes. I thought I was supposed to adapt the old one for this. I kept the first few lines with the course name and stuff because they seemed good and then just added the core curriculum stuff at the end because that was the only thing I could think of that needed to be added at course level.

Reply:
Yeah nothing wrong with using it as a base that's what I intended it for, just meant remove the stuff that isn't necessary, such as that comment :)